test/expected/sql/db-analysis-output/mssql/addresses.json

```diff
@@ -1,35 +1,41 @@
 {
   "fields": [{
     "name": "userId",
-    "nameColumn": "user_id",
+    "columnName": "user_id",
     "type": "INTEGER",
     "primaryKey": true,
     "defaultValue": null
   }, {
     "name": "street",
```

Comment:
This comment is about all the test folder.
<img width="392" alt="Screenshot 2019-12-11 at 12 11 35" src="https://user-images.githubusercontent.com/3200482/70616839-a6fe6b00-1c0f-11ea-948d-707195f5d67d.png">

The folders should be refactored in order to keep it simple: 
`expected` folder should contain => mongo, mssql, mysql, postgres and that's all. 

To obtain that, this actions are required:
sql/db-analysis-output/xxx => merge into expected/xxx (take care about name collision)
sql/foreign-keys-analysis-output/xxx => split files by db and merge into expected/xxx 
sql/dumper-output is unused => to delete
sql/renderings-sequelize.json is unused => to delete

After that, each DB will have its own expected folder 👍